DigitalLearningSolutions.Data/Models/CustomPrompts/CentreRegistrationPrompt.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
+{
+    using System;
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Enums;
+
+    public class CentreRegistrationPrompt : Prompt
+    {
+        public CentreRegistrationPrompt(RegistrationField registrationField, string text, string? options, bool mandatory)
+        {
+            RegistrationField = registrationField;
+            PromptText = text;
+            Options = SplitOptionsString(options);

```

Comment:
Why not set PromptText and Options in Prompt?

Reply:
moved to Prompt